salt/modules/boto_rds.py

```diff
@@ -447,6 +474,46 @@ def create_parameter_group(name, db_parameter_group_family, description,
         return {'error': salt.utils.boto3.get_error(e)}
 
 
+def create_db_cluster(name, engine, master_username, master_user_password,
+                      tags=None, region=None, key=None, keyid=None,
+                      profile=None):
+    '''
+    .. versionadded:: Fluorine
```

Comment:
I'm also thinking some doc strings describing the input options (I very often just crib the info from the boto docu, changing where needed to make it sensible in the Salt context).